style.css

```diff
@@ -0,0 +1,119 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body {
+  min-width: 400px;
+  min-height: 100vh;
+
+  display: grid;
```

Comment:
remove grid from here because you override it by display flex

Reply:
changes made please check @ManSOorcode